build_misc/base-files/README.FHS

```diff
@@ -0,0 +1,8 @@
+Due to *OS limitations, FHS cannot be followed strictly.
+
+Therefore, on iOS, tvOS, bridgeOS, audioOS and watchOS of Darwin version <= 20, some
+packages install files to /usr/local.
+
+On macOS, everything is prefixed in /opt/procursus
+and on iOS, tvOS, bridgeOS, audioOS and watchOS of darwin version >= 21,
+everything is prefixed in /private/preboot/procursus.
```

Comment:
Did I miss something? What's `/private/preboot/procursus`?

Reply:
this is the proposed location of the iOS 15 bootstrap. @Diatrus  agrees

Reply:
Cause of iOS' limitations on executable locations I presume?

Reply:
No, it is due to sealed system volume.

Reply:
Well it's partially because of executable locations and so it's not in /var

Reply:
Ok, seems nice, would `/opt/procursus` not work? Perhaps mount a image on `/opt` so that you can write to it. Then multiarch would actually be useful across iOS and macOS. I'm just spitballing here though.

Reply:
/opt doesn't exist in stock iOS, and union mount is gone too.

Reply:
It doesn't exist so a union mount would be unnecessary.

Reply:
???????

You need it to exist to mount over it

Reply:
Oh, sorry, I misunderstood.